setup.cfg

```diff
@@ -12,8 +12,8 @@ all_files  = 1
 [upload_sphinx]
 upload-dir = docs/build/html
 
-[pep8]
-ignore = E221
+[pycodestyle]
```

Comment:
pycodestyle 2.0 (2016-05-31) is also pycodestyle, but it does not recognize the `[pycodestyle]` section in `setup.cfg`. This has been changed in [2.1.0 (2016-11-04)](https://pypi.python.org/pypi/pycodestyle#id4). So for distros not newer than some months, the settings would not apply. I duplicated the section to keep support for these versions. See 59246fbd5f1636797697b1676e08eceb3475ac16